Release notes for 5.16.1 (#2032)
sensu/sensu-docs: content/sensu-go/5.16/release-notes.md

---

## 5.16.1 release notes

**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.
This is a patch release that optimizes the function the entity limiter uses.

See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.

> 

Author: ```suggestion

**IMPROVEMENTS**

- The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.

```

---

## 5.16.1 release notes

**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.

> This is a patch release that fixes a performance issue ...

Author: ```suggestion
This is a patch release that optimizes the function the entity limiter uses.
```

See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.

**FIXES:**

> - ...

Author: ```suggestion
- Fixed a performance regression introduced in 5.15.0 that caused the API to timeout past 20k agent sessions.
```

> - ...

Author: ```suggestion
```

---

## 5.16.1 release notes

**December 18, 2019** &mdash; The latest release of Sensu Go, version 5.16.1, is now available for download.

> This is a patch release that optimizes the function the entity limiter uses.

Author: ```suggestion
This is release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```

> This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.

Reviewer: ```suggestion
This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` prometheus counter by default.
```

> This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.

Author: ```suggestion
This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` counter by default.
```

> This is release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.

Author: ```suggestion
This release fixes a performance regression that caused the API to time out past 20k agent sessions and includes a change to display the `sensu_go_events_processed` counter by default.
```

> This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` counter by default.

Author: ```suggestion
This release fixes a performance regression that caused API latency to scale linearly as the number of connected agents increased and includes a change to display the `sensu_go_events_processed` Prometheus counter by default.
```

See the [upgrade guide][1] to upgrade Sensu to version 5.16.1.

**IMPROVEMENTS**

> - The `sensu_go_events_processed` counter now initializes with the `success` label so the count is always displayed.

Reviewer: ```suggestion
- The `sensu_go_events_processed` prometheus counter now initializes with the `success` label so the count is always displayed.
```

> - The `sensu_go_events_processed` prometheus counter now initializes with the `success` label so the count is always displayed.

Author: ```suggestion
- The `sensu_go_events_processed` Prometheus counter now initializes with the `success` label so the count is always displayed.
```

**FIXES:**

> - Fixed a performance regression introduced in 5.15.0 that caused the API to timeout past 20k agent sessions.

Author: ```suggestion
- The performance regression introduced in 5.15.0 that caused the API to time out past 20k agent sessions is now fixed.
```

> - The performance regression introduced in 5.15.0 that caused the API to time out past 20k agent sessions is now fixed.

Author: ```suggestion
- The performance regression introduced in 5.15.0 that caused API latency to scale linearly as the number of connected agents increased is fixed.
```
